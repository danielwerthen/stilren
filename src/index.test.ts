import { createTransformer } from "./index";

describe("Transform", () => {
  const transform = createTransformer({});
  it("should just work", () => {
    expect(
      transform({
        fontSize: "66px",
        smallMediumFontSize: "12px"
      })
    ).toEqual(4);
  });
});
