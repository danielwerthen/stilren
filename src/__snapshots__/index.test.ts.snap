// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Transform should just work 1`] = `
"<div class=\\"style={
  &#x27;@media (max-width: 768px)&#x27;: {
    color: &#x27;blue&#x27;
  },
  &#x27;@media (max-width: 768px) or (min-width: 1025px)&#x27;: {
    &#x27;:hover&#x27;: {
      backgroundColor: &#x27;yellow&#x27;
    }
  },
  color: &#x27;red&#x27;
}
\\">Test</div>"
`;
